app/keepers.go

```diff
@@ -103,6 +104,7 @@ type appKeepers struct {
 	MintKeeper           *mintkeeper.Keeper
 	PoolIncentivesKeeper *poolincentiveskeeper.Keeper
 	TxFeesKeeper         *txfeeskeeper.Keeper
+	FeeGrantKeeper       *feegrantkeeper.Keeper
```

Comment:
This PR is adding the FeeGrant module to Osmosis. I'm no longer opposed to adding it (I was last release, due to time since last security bug in the module, but a bit confusing why its bundled in this PR vs discussed elsewhere

Reply:
I would prefer we separate it out into a second issue to discuss its inclusion / if its what we want.

Reply:
Lets handle this decision in a second PR